website/docs/jellyfish/api/loan.md

```diff
@@ -277,4 +277,32 @@ interface UTXO {
   txid: string
   vout: number
 }
+```
+
+## getVault
+
+Returns information about vault.
+
+```ts title="client.loan.createVault()"
+interface loan {
+  getVault (vaultId: string): Promise<VaultDetails>
+}
+
+interface VaultDetails {
+  loanSchemeId: string
+  ownerAddress: string
+  isUnderLiquidation: boolean
+  batches?: AuctionBatchDetails[]
+  collateralAmounts?: string[]
+  loanAmount?: string[]
+  collateralValue?: BigNumber
+  loanValue?: BigNumber
+  currentRatio?: number
```

Comment:
Should be BigNumber

Reply:
it's an `int`.
https://github.com/DeFiCh/ain/blob/846cfa8f1b87619739c6f01b13f5e506341e7822/src/masternodes/rpc_vault.cpp#L68

Reply:
@jingyi2811 will change it to `BigNumber` since we fetch it as `BigNumber`. i.e `return await this.client.call('getvault', [vaultId], 'bignumber')`
nice catch 👏

Reply:
or `this.client.call('getvault', [vaultId], { collateralValue: BigNumber, loanValue: BigNumber })` to keep `currentRatio` as number same typing as `ain`

Reply:
@surangap 
make use of jellyfish-json
can try this
```ts
this.client.call('getvault', [vaultId], { currentRatio: number })
```